src/components/ArticleCard/ArticleCard.jsx

```diff
@@ -1,17 +1,23 @@
+import { Link } from "react-router-dom"
 import './ArticleCard.css'
-const ArticleCard = ({ title, topic, created_at, article_img_url, author }) => {
+
+const ArticleCard = ({ id, title, topic, created_at, article_img_url, author }) => {
   const dateString = new Date(created_at).toLocaleString('default', {month: 'short', day: 'numeric', year: 'numeric'})
 
+  const articlePath = `/${encodeURIComponent(title.toLowerCase().replaceAll(' ', '-'))}-${id}`
```

Comment:
I think this is very unnessary and needs revision.

Reply:
I feel like including the titile of the article in the URL makes it clearer for the user what site they are entering and what content they're about to see before clicking the link if it was let's say, shared with them. Would separating article_id from title with a slash be a better way to do it?
```suggestion
  const articlePath = `/articles/${encodeURIComponent(title.toLowerCase().replaceAll(/[ .]/g, '-'))}/${id}`
```